### PR TITLE
Citeseq celltypes

### PR DESCRIPTION
**Pull request by drighelli**
adding code, metadata, script, new functions and ontomap for the cord_blood identified celltypes.

### CODE REVIEW

**Comment by drighelli** [comment]
There was a conflict that has been resolved by adding the new code in the `.cord_blood` function.
It solves a bug where the cd could be also an empty object.

File: inst/extdata/metadata.csv [hunk 1]
```diff
@@ -40,6 +40,7 @@ mouse_visual_cortex,seqFISH_Coordinates.rda,SingleCellMultiModal/mouse_visual_co
...
 mouse_visual_cortex,seqFISH_Labels.rda,SingleCellMultiModal/mouse_visual_cortex/v2.0.0/seqFISH_Labels.rda,Rda,data.frame,Dario Righelli <dario.righelli@gmail.com>,"Dept. of Molecular Genetics, Allen Institute for Brain Science, United States",NA,10090,Mus musculus,2.0.0,https://www.dropbox.com/sh/avj4nrd4la5i88u/AACafWwBbE-xsLvOGDwRZDpYa?dl=0,TXT,NA,3.12,seqFISH_Labels data specific to the MOUSE_VISUAL_CORTEX project,seqFISH_Labels
 cord_blood,scADT_Counts.rda,SingleCellMultiModal/cord_blood/v1.0.0/scADT_Counts.rda,Rda,matrix,Dario Righelli <dario.righelli@gmail.com>,"Innovation Lab, New York Genome Center, New York, United States",NA,9606,Homo sapiens,1.0.0,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE100866,TXT,NA,3.12,scADT_Counts data specific to the CORD_BLOOD project,scADT_Counts
+cord_blood,scADT_clrCounts.rda,SingleCellMultiModal/cord_blood/v1.0.0/scADT_clrCounts.rda,Rda,matrix,Dario Righelli <dario.righelli@gmail.com>,"Innovation Lab, New York Genome Center, New York, United States",NA,9606,Homo sapiens,1.0.0,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE100866,TXT,NA,3.12,scADT_clrCounts data specific to the CORD_BLOOD project,scADT_clrCounts
```
Comment (LiNk-NY) [inline inst/extdata/metadata.csv:43]: (for future reference) please add to the bottom of the file

**Review by LiNk-NY — APPROVED** [review]
It looks good. I will move the metadata entry to the end of the `metadata.csv`.

**Comment by LiNk-NY** [comment]
Please try to avoid merge commits with [devel](https://github.com/waldronlab/SingleCellMultiModal/pull/69/commits/509d4ca3574de0dabae5cc9dc21761ef33f8fd43) and instead checkout from `devel` and commit your changes on top of the branch.
That is:
```
git checkout -b new_branch origin/devel
git commit # etc.
git push origin new_branch
```

**Comment by LiNk-NY** [comment]
Merged with rebase to clean up the merge commit. Thanks!

**Comment by LiNk-NY** [comment]
@drighelli 
Note. Make sure that the `BiocVersion` matches the current devel version `3.18` in the `metadata.csv`.

I have updated this.

**Comment by drighelli** [comment]
ok, sorry for the errors!

**Pull request closed without merging**